lib/msal-angular/src/msal.service.ts

```diff
@@ -56,7 +56,7 @@ export class MsalService extends UserAgentApplication {
                 this.setloginInProgress(false);
             }
             else if (this.getAcquireTokenInProgress()) {
-                broadcastService.broadcast("msal:acquireTokenFailure", msalError);
+                broadcastService.broadcast("msal:acquireToken", msalError);
```

Comment:
Why was this changed?

Reply:
Not sure. I have reverted this change, may have occurred due to an errant merge resolution.